fem/intrules_cut.hpp

```diff
@@ -18,6 +18,11 @@
 #include "eltrans.hpp"
 #include "coefficient.hpp"
 
+
+#ifdef MFEM_USE_ALGOIM
+#include <algoim_quad.hpp>
```

Comment:
Right now I get a ton of warnings when I build mfem due to blitz:

```
warning: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead
```

We could do the following to prevent that:
```
#ifdef MFEM_USE_ALGOIM
#ifdef MFEM_HAVE_GCC_PRAGMA_DIAGNOSTIC
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
#endif
#include <algoim_quad.hpp>
#pragma GCC diagnostic pop
#endif
```

Reply:
corrected - however I cannot test it on my machine - let me know once you can test it